numba/unicode.py

```diff
@@ -570,6 +570,27 @@ def split_whitespace_impl(a, sep=None, maxsplit=-1):
         return split_whitespace_impl
 
 
+@overload_method(types.UnicodeType, 'ljust')
+def unicode_ljust(string, width, fillchar=' '):
+    if not isinstance(width, types.Integer):
+        raise TypingError('<width> must be an Integer')
```

Comment:
I'm not sure why the unit tests are passing, since TypingError was not imported at the top of the file.  (this issue was found by flake8)

In any event, you should import TypingError.  :)

Reply:
@seibert I already put this type import in PR #3978 
I expect all tests pass after 3978 merge.